### PR TITLE
feat(content): update the intro-to-project part

### PR DESCRIPTION
**Pull request by jlevesy**
### What does this PR do?

This PR refreshes the `introduction to project` part.

### CODE REVIEW

File: content/chapitres/introduction-to-project.adoc [hunk 1]
```diff
@@ -1,33 +1,44 @@
...
+** 🔒 Vous vérouillez et laissez le véhicule sur votre lieu d'arrivée
+** 💸 Vous payez le temps passé sur le véhicule
 
-Expliquer l'app
-Expliquer la fonctionnalite qu'on implemente
-Trouver une excuse pour justifier le fait qu'on donne une tarball.
+== Contexte(2/4)
 
-== Prise en Main du Projet (1/2)
+Un cas d'utilisation principaux est de permettre aux utilisateurs de trouver un véhicle proche d'eux facilement.
```
Comment (dduportal) [inline content/chapitres/introduction-to-project.adoc]:
```suggestion
Un cas d'utilisation majeur est de permettre aux utilisateurs de trouver un véhicule proche d'eux facilement.
```

ou

```suggestion
Un cas d'utilisation principal est de permettre aux utilisateurs de trouver un véhicule proche d'eux facilement.
```

File: content/chapitres/introduction-to-project.adoc [hunk 2]
```diff
@@ -1,33 +1,44 @@
...
 
-* Une équipe technique de **Bananes** avait commencé l'implémentation du serveur, et à fourni une archive link:media/menu-server.tar.gz[téléchargeable ICI], contenant le code source du projet
-* **Bananes** vous assure qu'ils ont suivi toutes les "best practices" du développement logiciel sur minitel
-** Il y à un `LISEZMOI.txt` à la racine du projet :tada:
-** ... et pas grand chose d'autre?
+image::voimap.jpeg[caption="Voimap",width=256]
+
+== Contexte(3/4)
+
+* Ce service est assuré par une API HTTP appellée `vehicle-server` qui doit supporter fonctionalités suivantes:
```
Comment (dduportal) [inline content/chapitres/introduction-to-project.adoc]:
```suggestion
* Ce service est assuré par une API HTTP appellée `vehicle-server` qui doit supporter les fonctionalités suivantes:
```

File: content/chapitres/introduction-to-project.adoc [hunk 3]
```diff
@@ -1,33 +1,44 @@
...
+** Enregistrer un véhicle
+** Trouver les N véhicules les plus proches de soi
+** Supprimer un véhicule.
+
+== Contexte (4/4)
+
+* Voi est entrain de reconstruire cette fonctionalité et à décidé de sous-traiter le développement de ce projet a l'ENSG...
```
Comment (dduportal) [inline content/chapitres/introduction-to-project.adoc]:
```suggestion
* Voi est entrain de reconstruire cette fonctionalitée et à décidé de sous-traiter le développement de ce projet a l'ENSG...
```

File: content/chapitres/introduction-to-project.adoc [hunk 4]
```diff
@@ -38,7 +49,7 @@ A partir de la vous pouvez ouvrir le fichier `LISEZMOI.txt` et commencer à suiv
 == Triste Rencontre avec la Réalité
 
 - Pas de gestion de version...
-- Le projet ne fonctionne pas, tous les menus retournés s'appellent "TODO" :sob:
+- Le projet ne fonctionne pas, tous les vehicles répondus s'appellent "TODO" :sob:
```
Comment (dduportal) [inline content/chapitres/introduction-to-project.adoc]:
```suggestion
- Le projet ne fonctionne pas, tous les véhicules répondus s'appellent "TODO" :sob:
```

**Review by dduportal — APPROVED** [review]
(nit) A few typos other than that, noice!

File: content/chapitres/introduction-to-project.adoc (hunk 1 above)
Comment (jlevesy) [reply content/chapitres/introduction-to-project.adoc]: I don't know how to french anymore, thanks :D

**Pull request merged**